modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
 import {CSSObject, jsx} from '@emotion/core';
 import {accessibleHide, GrowthBehavior} from '@workday/canvas-kit-react-common';
 import {depth, spacing, commonColors, borderRadius} from '@workday/canvas-kit-react-core';
 import {MenuItemProps} from '@workday/canvas-kit-labs-react-menu';
+import {styled, Themeable, ContentDirection} from '@workday/canvas-kit-labs-react-core';
```

Comment:
This import violates the rule where labs components can import from production or other labs components, but production can't import from labs. This might be necessary in this case, especially if the prod APIs don't change with the changes to the labs components/utilities that are relied on. In this case, `theme` might have to be an unstable API inside a stable API.

Reply:
Yep - if we want to support it for testing purposes (or to quietly add support to all of our components while we finalize the theme object/API), we need to break this rule in this case.